src/gpl/test/simple02.py

```diff
@@ -10,9 +10,11 @@
 design = helpers.make_design(tech)
 design.readDef("./simple02.def")
 
-gpl_aux.global_placement(
-    design, density=0.6, init_density_penalty=0.01, skip_initial_place=True
-)
+options = gpl.PlaceOptions()
+options.initDensityPenaltyFactor = 0.01
+options.density = 0.6
+options.initialPlaceMaxIter = 0
```

Comment:
Is it possible / reasonable to have the options object constructed with keyword args, rather than setting individual members after? e.g.
```
options = gpl.PlaceOptions(initDensityPenaltyFactor = 0.01,
                           density = 0.6,
                           initialPlaceMaxIter = 0)
```

Reply:
That would be great but I'm struggling just to get cmake & bazel to both work.  I'll look at that later.  I want to avoid having to write a wrapper method for every class.